index.html

```diff
@@ -0,0 +1,296 @@
+              <span class="subheading-list-title">2015</span>
+            </div>
+            <section class="card-content">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </section>
+            <ul class="webdev-languages">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+            <button class="see-project" type="button">See project</button>
+          </div>
+        </div>
+        <div class="cards">
+          <img
+            src="assets/snapshoots/Snapshoot Portfolio-3.svg"
+            alt="Snapshoot svg"
+          />
+          <div class="cards-description">
+            <h2 class="card-titles">Tonic</h2>
+            <div class="subheading-lists">
+              <span class="canopy">CANOPY</span>
+              <img src="assets/icons/circle-icon.svg" alt="circlular icon" />
+              <span class="subheading-list-title">Back End Dev</span>
+              <img src="assets/icons/circle-icon.svg" alt="circlular icon" />
+              <span class="subheading-list-title">2015</span>
+            </div>
+            <section class="card-content">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </section>
+            <ul class="webdev-languages">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+            <button class="see-project" type="button">See project</button>
+          </div>
+        </div>
+        <div class="cards">
+          <img
+            src="assets/snapshoots/Snapshoot Portfolio-4.svg"
+            alt="Snapshoot svg"
+          />
+          <div class="cards-description">
+            <h2 class="card-titles">
+              Multi-Post <br />
+              Stories
+            </h2>
+            <div class="subheading-lists">
+              <span class="canopy">CANOPY</span>
+              <img src="assets/icons/circle-icon.svg" alt="circlular icon" />
+              <span class="subheading-list-title">Back End Dev</span>
+              <img src="assets/icons/circle-icon.svg" alt="circlular icon" />
+              <span class="subheading-list-title">2015</span>
+            </div>
+            <section class="card-content">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </section>
+            <ul class="webdev-languages">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+            <button class="see-project" type="button">See project</button>
+          </div>
+        </div>
+      </section>
+      <section class="about-myself">
```

Comment:
- [ ] You did a great job with semantics and design. :+1: Let's fix some minor issues to achieve the project requirements. 🤝️
  - I think the spacing on the left side of this entire section is bigger when compared to the design. Kindly reduce it a bit. 🥸️
  - Kindly put the text `About Myself` on a single line to match the design. 🙂️ 
  - Please align the left side of the button `Get my resume` with the icons above. 🧐️ 
  - The arrows are not aligned properly which inturn created a huge space on the right side of screen throughout the whole page. Kindly remove the space created on the right because it is making the design look a bit unprofessional. ✍️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/235309468-5bd57ec1-9738-4bf4-9bc5-e4d52c1136d8.png" width="300"><img src="https://user-images.githubusercontent.com/17243325/235309470-42f35290-0071-48ec-973f-1ba2873c55c7.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/235309464-a1f52334-9405-4219-99c7-ca53c2148794.png" width="300">|